src/App.tsx

```diff
@@ -17,33 +19,36 @@ const App: React.FC = () => {
   );
 
   return (
-    <Router>
-      <AppStateProvider>
-        <InstantSearch searchClient={searchClient} indexName={config.algoliaIndexName}>
-          <Configure hitsPerPage={8}/>
-          <div className="app">
-          <header className="app__header">
-            <AppHeader />
-          </header>
-          <main className="app__main">
-            <Switch>
-              {routes.map(route => (
-                <Route key={route.path} {...route} />
-              ))}
-            </Switch>
-          </main>
-          <footer id="app-footer" role="contentinfo" aria-label="app-footer">
-            <div className="app__footer">
-              <AppFooter />
-            </div>
-          </footer>
-          <aside className="app__compareoverlay">
-            <CompareOverlay />
-          </aside>
-        </div>
-        </InstantSearch>
-      </AppStateProvider>
-    </Router>
+    <APIErrorProvider>
```

Comment:
Any concern with the error provider being outside our router? Could it impact our routing? Not sure, just generally asking.

Reply:
It shouldn't impact our routing. But I can move it inside the router